### ISSUE TITLE
Returned shape does not match documentation

### ISSUE THREAD

**Issue by nvssynthesis** [maxrmorrison/torchcrepe#42]
There are many scenarios where the returned shape of torchcrepe.predict does not match the documentation. I cannot figure out what the pattern is without digging into the implementation. Below is an example where I don't understand what determines the final number of frames, whether or not it's using padding. 

Can you please explain what is determining the shape, and update the documentation?

`import torch`
`import torchcrepe`

`N=678987`
`h=100`
`x = torch.rand([1, N])`
`expected_num_frames = 1 + int(N // h)`
`print(expected_num_frames) # prints 6790`

`p, v = torchcrepe.predict(x, sample_rate=22050, 
                                               hop_length=h, 
                                                model='tiny', decoder = torchcrepe.decode.viterbi, 
                                                return_periodicity = True, 
                                                batch_size=256, device='mps',
                                                pad=True) # yes padding`
`print(p.shape == expected_num_frames) # false`
`print(p.shape) # prints "torch.Size([1, 6843])"`

`p, v = torchcrepe.predict(x, sample_rate=22050, 
                                               hop_length=h, 
                                                model='tiny', decoder = torchcrepe.decode.viterbi, 
                                                return_periodicity = True, 
                                                batch_size=256, device='mps',
                                                pad=False) # no padding` 
`print(p.shape == expected_num_frames) # false`
`print(p.shape) # prints "torch.Size([1, 6829])"`

**Comment by maxrmorrison** [comment]
Pitch estimation occurs at 16 khz, not 22.05 khz. You should update your hopsize accordingly. This is also why my follow-up estimator, [`penn`](https://github.com/interactiveaudiolab/penn), uses a fractional hopsize in seconds.

**Comment by maxrmorrison** [comment]
See also the first torchcrepe usage example, where the hopsize is computed using the audio sample rate.
